data/profiles/install-centos.ipxe

```diff
@@ -3,5 +3,10 @@ set base-url <%=repo%>/images/pxeboot
 set params initrd=initrd.img ks=<%=installScriptUri%> hostname=<%=hostname%> ksdevice=bootif BOOTIF=01-${netX/mac} console=<%=comport%>,115200n8 console=tty0
 kernel ${base-url}/vmlinuz repo=<%=repo%> ${params}
 initrd ${base-url}/initrd.img
+
+# Below two lines are designed for progress info, downloading action itself is meaningless.
+initrd http://<%=server%>:<%=port%>/api/current/notification/<%=taskId%>/<%=steps%>
```

Comment:
I understand that's no CURL/wget tool for accessing an URL in iPXE environment, but use ```imgfetch``` instead of ```initrd``` to access url and using ```--name uselessoutput``` may help make the code more readable.